### DISCUSSION TITLE
Disable vcpkg on this Project

### DISCUSSION THREAD

**Discussion by Sarund9** [premake/premake-core#1812] [Q&A]
I have [vcpkg](https://github.com/microsoft/vcpkg) installed and integrated into vs2019 with `vcpkg integrate install`
I would like to disable it on a particular project (It causes issues with Clang which I am using)
As specified in [this Issue](https://github.com/microsoft/vcpkg/issues/281)

My question is how do I add the `/p:VcpkgEnabled=false` to every project in my workspace

**Comment by samsinsane** [comment]
It would depend on whether that's a compile option, linker option or both.
- Compile option: `buildoptions { "/p:VcpkgEnabled=false" }`
- Linker option: `linkoptions { "/p:VcpkgEnabled=false" }`
